app/src/main/res/xml/preferences.xml

```diff
@@ -5,11 +5,13 @@
     <fr.free.nrw.commons.ui.LongTitlePreferences.LongTitlePreferenceCategory
         android:title="@string/preference_category_appearance">
 
-        <fr.free.nrw.commons.ui.LongTitlePreferences.LongTitleSwitchPreference
-            android:title="@string/preference_theme"
-            android:defaultValue="false"
-            android:summary="@string/preference_theme_summary"
-            android:key="theme" />
+        <fr.free.nrw.commons.ui.LongTitlePreferences.LongTitleListPreference
+            android:key="themePref"
+            android:title= "@string/preference_theme"
+            android:defaultValue="@string/theme_default_name"
+            android:entries="@array/pref_theme_entries"
+            android:entryValues="@array/pref_theme_entries"
```

Comment:
I am uncomfortable using a potentially translated string to represent the underlying value. If the user changes language nne of these comparisons will be accurate for the stored value. I'd use numbers 0..2

Reply:
I understood your point, changes made thank you :)